cosinnus/api_frontend/views/navigation.py

```diff
@@ -621,14 +621,15 @@ class LanguageMenuItemMixin:
     def get_language_menu_item(self, request, current_language_as_label=False):
         language_item_label = request.LANGUAGE_CODE.upper() if current_language_as_label else _('Change Language')
         language_item_icon = None if current_language_as_label else 'fa-language'
-        language_item = MenuItem(language_item_label, None, language_item_icon, id='ChangeLanguage')
+        language_item = MenuItem(language_item_label, icon=language_item_icon, id='ChangeLanguage')
         language_selection = filter(lambda l: l[0] in settings.COSINNUS_V3_FRONTEND_SUPPORTED_LANGUAGES,
                                     settings.LANGUAGES)
-        language_subitems = [
-            MenuItem(language, reverse('cosinnus:switch-language', kwargs={'language': code}),
-                     id=f'ChangeLanguageItem{code.upper()}')
-            for code, language in language_selection
-        ]
+        language_subitems = []
+        for code, language in language_selection:
+            language_subitem = MenuItem(language, reverse('cosinnus:switch-language', kwargs={'language': code}),
+                                        id=f'ChangeLanguageItem{code.upper()}')
+            language_subitem['selected'] = code == request.LANGUAGE_CODE
```

Comment:
@2e2a the `selected` property in `MenuItem` is something agreed upon with the frontend, right? In that case, we should add that definition to `MenuItem`'s signature, as a parameter, so it is clear that this is an existing property.